core/job/handler/v1beta1/job.go

```diff
@@ -45,6 +45,7 @@ type JobService interface {
 	Add(ctx context.Context, jobTenant tenant.Tenant, jobs []*job.Spec) error
 	Update(ctx context.Context, jobTenant tenant.Tenant, jobs []*job.Spec) error
 	UpdateState(ctx context.Context, jobTenant tenant.Tenant, jobNames []*job.Name, jobState job.State, remark string) error
+	SyncState(ctx context.Context, jobTenant tenant.Tenant, disabledJobs, enabledJobs []*job.Name) error
```

Comment:
just like the previous PR feedback, we can remove the pointer for `job.Name`

Reply:
sure